oneandone/src/main/java/org/apache/jclouds/oneandone/rest/domain/Types.java

```diff
@@ -202,4 +202,20 @@ public static VPNType fromValue(String v) {
          return Enums.getIfPresent(VPNType.class, v).or(UNRECOGNIZED);
       }
    }
+
+   public enum PeriodType {
+      LAST_HOUR, LAST_24H, LAST_7D, LAST_30D, LAST_365D, UNRECOGNIZED;
+
+      public static PeriodType fromValue(String v) {
+         return Enums.getIfPresent(PeriodType.class, v).or(UNRECOGNIZED);
+      }
+   }
+
+   public enum CustomPeriodType {
+      CUSTOM, UNRECOGNIZED;
```

Comment:
i intentionally separated them, when you choose CUSTOM you must provide start and end date all the other types do not require that, easier for users.